corehq/ex-submodules/couchexport/tests/test_schema.py

```diff
@@ -110,7 +110,7 @@ def test_split_column_ignore_extras(self):
 
     def test_split_column_missing(self):
         self.post_it()
-        self._test_split_column([[None, None, None, None, None]])
+        self._test_split_column([[None, None, None, None, SCALAR_NEVER_WAS]])
```

Comment:
heh, guess i was wrong. updated and they pass locally now (see last commit for another test i had to update).

just for my own understanding, can you explain why that is? i was having trouble tracing what was happening